e2e/_suites/fleet/stand-alone.go

```diff
@@ -87,7 +87,7 @@ func (sats *StandAloneTestSuite) theStandaloneAgentIsListedInFleetWithStatus(des
 	return nil
 }
 
-func (sats *StandAloneTestSuite) aStandaloneAgentIsDeployedWithFleetServerMode(image string) error {
+func (sats *StandAloneTestSuite) bootstrapFleetServerFromAStandaloneAgent(image string) error {
```

Comment:
Nit: This has a capitalization mismatch. In `StandAloneTestSuite` the word `Alone` is capitalized by in `bootstrapFleetServerFromAStandaloneAgent`, it is lowercase.

Reply:
What is the correct spell for standalone? I think the wrong one is the `StandAloneTestSuite` right?